### PR TITLE
Pushing all databases interactions with dashboard

### PR DESCRIPTION
**Pull request by lepatrick714**

### CODE REVIEW

File: my-app/src/App.js [hunk 1]
```diff
@@ -7,7 +7,44 @@ import { Login } from './Login.js'
...
+  constructor(props){
+    super(props);
+    //this.app = firebase.initializeApp(fbconfig);
+
+
+
+    this.db = firebase.database().ref().child('userFrame');
+  }
+
+/* From Youtube tutorial - complex method??
```
Comment (jonei005) [inline my-app/src/App.js]: Is this comment going to be used in the future? Or can you remove it?

**Comment by jonei005** [comment]
This is the same branch that includes the earlier changes of adding to database right? So we should be able to create and remove boards from our accounts now?

I checked out the branch and looks like adding/deleting both work well. I noticed a small bug in that I created 2 boards, then refreshed the page, and the order of the boards switched. Not important but maybe something to note. I didn't see that problem when Adrian was testing earlier so not sure what happened. 

Either way, that's not a huge bug, we can fix it later. I say try to clean up some of the comments and unnecessary console.logs(), then ship it.

File: my-app/src/App.js (hunk 1 above)
Comment (lepatrick714) [reply my-app/src/App.js]: Removed. I don't believe it does anything

**Comment by lepatrick714** [comment]
Yes the branch includes the earlier changes. I changed so that we push in our generated userUid instead of relying on push() and grabbing the key from that. 

This causes the database to sort the instances in abc order instead of what came in first and second. Therefore when we do a refresh to the page, it populates the page according to how it is in the db. It's a small little bug that might / will blow up. Definitely something to consider

I believe that I cleaned up many of the unnecessary console.logs() but do note them if you spot something important 

@jonei005

**Comment by lepatrick714** [comment]
Also going to leave this PR open until the rest of the team takes a look at it

**Comment by jonei005** [comment]
Interesting, okay! I was looking in the database and I think I'm confused what certain uid's are.

uid-question

In listOfBoards, "My First Board" is listed under the correct uid, then it has a userId (I assume is supposed to be an ID for my account). 

Then in listOfUsers, under my account (DZ6K3...) -> personalBoards, my board is listed under the same string that the userId is. I would think Boards should be stored under their own uid? Maybe you can explain to me.

**Comment by lepatrick714** [comment]
Aye it’s a very weird hack. I discussed this with my friend and he gave me this approach 

Essentially, we need to be able to reference the user’s board and the board itself. The userId represent the is in which the board reference lives in the user. Same goes for uid but on the user side. I suppose for the list of boards, we don’t have to have the uid since we could just refer to the object’s name instead

I can definitely explain this more throughly when we meet up again. As for now,, I would recommend adding boards, observe how their populated in the db, delete a board, observe what’s going on in the db, and finally tracing through the deleteboard() logic.

**Comment by jonei005** [comment]
@Kuzame @lepatrick714 

This is something I noticed about the database format, I think we should make it a little cleaner if possible. Basically in the picture I posted before, I noticed that "Syllm-OKkf" seemed unnecessary to me, we can use the actual board uid to identify it instead of having two IDs for the same board. Here's what I think would be cleaner:

firebase-new-format

If it's confusing I'll just explain in person when we meet.

**Pull request merged**